psbt_dump.py

```diff
@@ -241,13 +257,33 @@ def dump(psbt, hex_output, bin_output, testnet, base64, show_addrs):
                     print("'%s' = 0x%x = %d = %s\n" % (b2a_hex(val), nn, nn, SIGHASH_MAP[nn]))
                 else:
                     print("'%s' = 0x%x = %d\n" % (b2a_hex(val), nn, nn))
+
+                if key[0] == PSBT_GLOBAL_VERSION:
+                    print("\tPSBTv%d\n" % nn)
```

Comment:
track version of current file, show errors when incorrect fields vs. version are used